pkg/controllers/dynakube/injection/reconciler.go

```diff
@@ -79,6 +87,13 @@ func (r *reconciler) Reconcile(ctx context.Context) error {
 		setupErrors = append(setupErrors, err)
 	}
 
+	err := processmoduleconfigsecret.NewReconciler(
+		r.client, r.apiReader, r.dynatraceClient, r.dynakube, r.scheme, timeprovider.New()).
+		Reconcile(ctx)
+	if err != nil {
+		setupErrors = append(setupErrors, err)
+	}
+
```

Comment:
create the reconciler in the constructor, like the `versionReconciler`, so it can be mocked

also use the same `timeprovider` for the reconcilers (however I am not the biggest fan of using `.Freeze()` for no real reason)

Reply:
done in https://github.com/Dynatrace/dynatrace-operator/pull/2777/commits/39d3778db07cc9d89ba637c88bb091975411c30b